packages/server/src/restaurants/database/RestaurantSchema.ts

```diff
@@ -66,6 +67,22 @@ const RESTAURANT_CONSTANTS = Object.freeze({
   collection: 'restaurants',
 })
 class Restaurant {
+  get isCompleted() {
+    const requirements = {
+      name: this.name,
+      bankAccountNumber: this.bankAccountNumber,
+      phoneNumber: this.phoneNumber,
+      addressId: this.addressId,
+      logo: this.logo,
+      description: this.description,
+      cuisineType: this.cuisineType,
+      tags: this.tags,
+    };
```

Comment:
Jeśli korzystasz tylko i wyłącznie z wartości obiektu, to to równie dobrze może być tablica tych wartości.